configuration/pih/htmlforms/section-delivery.xml

```diff
@@ -702,49 +658,43 @@
                                 <obs conceptId="CIEL:5916" showUnits="true"/>
 
                                 <!-- APGAR -->
-                                <br/><br/>
+                                <br/>
+                                <br/>
                                 <label>
-                                    <uimessage code="pihcore.mch.apgarScores" />
```

Comment:
@Dutervil These are my recent changes.  You'll need to merge in the latest updates for the Apgar scores.  They are in the master branch.

Reply:
Ok, Yes

Reply:
I did